libdevcore/FixedHash.h

```diff
@@ -351,6 +351,25 @@ using h160Set = std::set<h160>;
 using h256Hash = std::unordered_set<h256>;
 using h160Hash = std::unordered_set<h160>;
 
+/// An Ethereum address: 20 bytes.
+/// @NOTE This is not endian-specific; it's just a bunch of bytes.
+using Address = h160;
```

Comment:
I'd prefer to put these into separate Address.h/cpp

Reply:
Two cons for this: increased build time (?) and additional maintenance cost. I'm not sure...

Reply:
I think it's your way it could be theoretically increased build time, because everyone who needs only `FixedHash` has to compile `Address`-related stuff, too

Reply:
It would make navigating the code easier, putting `Address` into Address.h instead of suprisingly `FixedHash.h`